packages/widgetbook_cli/bin/commands/publish.dart

```diff
@@ -436,62 +434,75 @@ class PublishCommand extends WidgetbookCommand {
 
       if (file != null) {
         progress.update('Uploading build');
-        final uploadInfo = await uploadDeploymentInfo(
+        final buildUploadResponse = await uploadDeploymentInfo(
           file: file,
           args: args,
         );
 
-        if (uploadInfo == null) {
+        if (buildUploadResponse == null) {
           throw WidgetbookApiException();
-        } else {
-          progress.complete('Uploaded build');
         }
 
-        // If generator is not run or not properly configured
-        if (useCases.isEmpty) {
-          logger.err(
-            'HINT: Could not find generator files. '
-            'Therefore, no review has been created. '
-            'Make sure to use widgetbook_generator and '
-            'run build_runner before this CLI. '
-            'See https://docs.widgetbook.io/widgetbook-cloud/review for more '
-            'information.',
+        for (final task in buildUploadResponse.tasks) {
+          if (task.status == UploadTaskStatus.success) {
+            logger.success('\n✅ ${task.message}');
+          } else if (task.status == UploadTaskStatus.warning) {
+            logger.info('\n⚠️ ${task.message}');
+          } else {
+            logger.err('\n❌ ${task.message}');
+          }
+        }
+
+        if (baseBranch == null) {
+          logger.info(
+            '💡Tip: you can upload a review by specifying a `base-branch` parameter.'
+            '\nSee https://docs.widgetbook.io/widgetbook-cloud/review '
+            'for more information',
```

Comment:
I would put the newline at the end of the line not at the beginning of another line